### PR TITLE
add-bastion-record

### PR DESCRIPTION
**Pull request by calvix**
add DNS record for bastion machine

### CODE REVIEW

**Comment by calvix** [comment]
towards https://github.com/giantswarm/roadmap/issues/1288

**Review by zewolfe — APPROVED** [review]
LGTM

File: controllers/awscluster_controller.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/pkg/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	capi "sigs.k8s.io/cluster-api/api/v1alpha3"
```
Comment (fiunchinho) [inline controllers/awscluster_controller.go:27]: could we use a more recent api version?

**Review by fiunchinho — COMMENTED** [review]
We should definitely have tests on this component. I think there are none at the moment, maybe we can drop a card in our board to bootstrap the tests here?

File: controllers/awscluster_controller.go (hunk 1 above)
Comment (calvix) [reply controllers/awscluster_controller.go:27]: I think we will need to do this for the whole controller, but that is out of the scope of this PR, 
bumping  the version but breaks the import and it wont build

I will drop a note to the board

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**